cla-backend-go/v2/dynamo_events/signatures.go

```diff
@@ -94,6 +94,59 @@ func (s *service) SignatureSignedEvent(event events.DynamoDBEventRecord) error {
 			log.WithFields(f).Warnf("failed to add signed_on date/time to signature, error: %+v", err)
 		}
 
+		// If CLA signature
+		if newSignature.SignatureType == CLASignatureType && newSignature.SignatureUserCompanyID != "" {
```

Comment:
use utils.CLASignatureType constant (replace the one in this package to use the utils/constants.go version).